tests/data/validate/validate.test.ts

```diff
@@ -0,0 +1,77 @@
+import Ajv from 'ajv';
+import { cloneDeep } from 'lodash';
+import * as metaschema from '../../../src/commands/data/schemas/util/metaschema.json';
+import { SchemaVerify, TestId } from '../../../src/commands/data/schemas/util/schemaverify';
+import { validSchema } from '../../__helpers__/schemas';
+
+describe('Data - Schema - Validate', () => {
```

Comment:
I would still add a test or two. One with an array in an array, because that is a case that is often missed when things are re-written. I missed it in my earlier PR and you did too in your first solution. And maybe one where to modifiers don't match.